packages/compass-components/.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  root: true,
+  parser: '@typescript-eslint/parser',
+  plugins: [
+    '@typescript-eslint',
+    'jsx-a11y'
+  ],
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended'
+  ],
+  env: { node: true },
+  overrides: [
```

Comment:
What do we do for packages using `mocha`? Should we go with `jest` everywhere we convert to `ts`? Maybe we could pull this file in `ROOT/config` have a `eslintrc.base.js` and then a `ROOT/config/eslintrc.jest.js` and `ROOT/config/eslintrc.mocha.js` extending it with relevant section for both?

If everyone is ok with moving to `jest` we could (and maybe should) create some action for that to avoid having different technologies all over.  There seem to be a `jest-codemods` we could experiment around with.

Reply:
I mean, I like mocha a lot, but yeah, it would be nice to have consistency across Compass.

Reply:
I slightly prefer mocha tbh, but no real stakes in there. I especially found it easier to debug / develop individual tests with and had some pain in stubbing things in Jest. But as @mcasimir said we'd definitely have to agree that all migrated TS code then uses only `jest`.

Reply:
Don't have a preference, from experience `jest` usually has more react-specific integrations available, e.g., `jsdom` env is included, tools like `testing-library` or `react-axe` usually come with a bunch of jest helpers already, whereas with `mocha` you'd usually need to configure it all yourself (but that's not really too critical, just a bit more maintenance on our side)

Reply:
Would be great to make a decision though .. I also don't have a preference.

Reply:
So just to outline:

`mocha`:

\+ preferred by more people in the team (or at least seems like it)
\+ we don't need to drastically rewrite other tests in the monorepo
\- a bit more work to set up in some cases

`jest`:

\+ better react integrations
\- nobody have a preference specifically for it
\- we will need to gradually migrate other tests to it, so additional work

seems like we have a clear winner here 😄

Reply:
Updated the tests to use mocha. 👍  Definitely fewer react integrations but we can work around that.